cl/lib/admin.py

```diff
@@ -13,3 +18,30 @@ class Media:
 class NotesInline(GenericTabularInline):
     model = Note
     extra = 1
+
+
+def build_admin_url(
+    model_class: Type[Model],
+    query_params: Optional[Dict[str, Any]] = None,
```

Comment:
whenever possible we try to use/upgrade to built-in types. For example, here we could use:

```suggestion
    query_params: dict[str, Any] | None = None,
```

other types that are not available like `Type` or `Any` it's ok to continue importing from the `typing` module.

Reply:
thank you for the comment, I'll keep that in mind! I've updated the code with your suggestion